### ISSUE TITLE
Create publication assets for IOS and Android

### ISSUE THREAD

**Issue by falconx** [neontribe/gbptm-ui#89]

**Comment by falconx** [comment]
Fixed. Added icon and splash screen assets.

Developer notes:
- Use [splashicon-generator](https://github.com/eberlitz/splashicon-generator)
- Useful for creating png from svg:
```
inkscape -z -e test.png -w 1024 -h 1024 test.svg
```

**Comment by falconx** [comment]
Had a lot of issues with Android.

1. Default splash needed to be added. Though it seems like this is the only getting being read:
```
<splash src="splash.png" />
```

2. There's a black/grey screen which appears before the splash (and sometimes after, before the "Fetching toilets..." screen). During this initial screen, a title bar was being displayed - I've managed to hide this with the following:
```
<config-file platform="android" parent="/manifest" mode="merge">
    <application android:theme="@android:style/Theme.NoTitleBar"></application>
</config-file>
```

3. It would be nice, instead of having the splash screen auto close, to programmatically close it once we've finished the initial data fetch. Unfortunately the following isn't working (at least, not reliably):
```
<preference name="AutoHideSplashScreen" value="false" />
```
